Datastore/tests/System/LookupTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Datastore\Tests\System;
+
+use Google\Cloud\Datastore\DatastoreClient;
+use Google\Protobuf\Timestamp;
+
+/**
+ * @group datastore
+ * @group datastore-save
+ */
+class LookupTest extends DatastoreMultipleDbTestCase
+{
+    /**
+     * @dataProvider defaultDbClientProvider
+     */
+    public function testLookupWithReadTime(DatastoreClient $client)
+    {
+        $kind = 'NewPerson';
+        $lastName = 'Geller';
+        $newLastName = 'Bing';
+        $key = $client->key($kind, time());
+        $person = $client->entity($key, [
+            'lastName' => $lastName
+        ]);
+        $client->insert($person);
+        self::$localDeletionQueue->add($key);
+
+        sleep(1);
```

Comment:
I observed little to no tests failures on PROD only when I do `sleep(2)` everywhere on this PR.

Reply:
I tried with 1 and 2s. Both worked for me actually. But if it is effecting you, I will change to sleep(2) everywhere.

Reply:
Done